### ISSUE TITLE
Neofetch launched through .bashrc prevent the right detection of node version

### ISSUE THREAD

**Issue by shakasan** [fwouts/previewjs#1261]
### Describe the bug

Neofetch launched through .bashrc prevent the right detection of node version.

We can see the returned version number just after the output of neofetch, but this prevent the right detection of the version by preview.js, thus, preview.js can't be started

Commenting the neofetch commande in .bashrc and restard vscode fix the issue.

### Reproduction

- install neofetch
- add the command (neofetch) in .bashrc
- open visual studio code
- the previewjs output report the error and can't be launched

### Preview.js version

v1.16.1

### Framework

Not linked to a framework

### System Info

```shell
System:
    OS: Linux 6.0 Manjaro Linux
    CPU: (12) x64 AMD Ryzen 5 3600 6-Core Processor
    Memory: 23.96 GB / 31.29 GB
    Container: Yes
    Shell: 5.1.16 - /bin/bash
  Binaries:
    Node: 16.18.0 - /usr/bin/node
    Yarn: 1.22.19 - /usr/bin/yarn
    npm: 9.1.2 - ~/.local/bin/npm
  IDEs:
    Nano: 6.4 - /usr/bin/nano
    VSCode: 1.73.1 - /usr/bin/code
  Browsers:
    Brave Browser: 107.1.45.133
    Chromium: 107.0.5304.121
    Firefox: 107.0
```

### Used Package Manager

npm

### Anything else?

![previews js_output](https://user-images.githubusercontent.com/16532111/204900623-55cc10f3-ba2a-4629-94cd-943639b33785.png)

**Comment by fwouts** [comment]
Thanks for the bug report! Let me see what we can do about this...

**Comment by fwouts** [comment]
This will be shipped hopefully next week :)

**Comment by shakasan** [comment]
Super, thanks for the help, I will test it as soon as it's available ;-)

**Comment by fwouts** [comment]
Hey @shakasan, this is now shipped in v1.17.0. Let me know how you go!

**Comment by shakasan** [comment]
> Hey @shakasan, this is now shipped in v1.17.0. Let me know how you go!

Just tested and working fine. Thanks a lot.
